platforms/core-runtime/launcher/src/main/resources/release-features.txt

```diff
@@ -0,0 +1,5 @@
+ - Configuration Cache is the preferred execution mode
```

Comment:
I know we've been saying "preferred mode of execution" for awhile, but maybe we should phrase this in a more user value way?  WDYT @bamboo 
e.g.,
```suggestion
 - Configuration Cache is recommended to be enabled for all builds
```